cumulus/polkadot-parachain/src/command.rs

```diff
@@ -596,31 +612,31 @@ pub fn run() -> Result<()> {
 				// that both file paths exist, the node will exit, as the user must decide (by
 				// deleting one path) the information that they want to use as their DB.
 				let old_name = match config.chain_spec.id() {
-					 "asset-hub-polkadot" => Some("statemint"),
-				     "asset-hub-kusama" => Some("statemine"),
-				     "asset-hub-westend" => Some("westmint"),
-				     "asset-hub-rococo" => Some("rockmine"),
-				     _ => None,
+					"asset-hub-polkadot" => Some("statemint"),
+					"asset-hub-kusama" => Some("statemine"),
+					"asset-hub-westend" => Some("westmint"),
+					"asset-hub-rococo" => Some("rockmine"),
+					_ => None,
 				};
 
 				if let Some(old_name) = old_name {
-				    let new_path = config.base_path.config_dir(config.chain_spec.id());
-				    let old_path = config.base_path.config_dir(old_name);
+					let new_path = config.base_path.config_dir(config.chain_spec.id());
+					let old_path = config.base_path.config_dir(old_name);
 
-				    if old_path.exists() && new_path.exists() {
-				         return Err(format!(
+					if old_path.exists() && new_path.exists() {
+							return Err(format!(
```

Comment:
Why does this indent two times more than the `if`? This particular section of code seems like the nemesis of `cargo fmt` :see_no_evil: 
```suggestion
						return Err(format!(
```